eip-template.md

```diff
@@ -1,47 +1,83 @@
 ---
-eip: <to be assigned>
 title: <The EIP title is a few words, not a complete sentence>
 description: <Description is one full (short) sentence>
 author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
 discussions-to: <URL>
 status: Draft
 type: <Standards Track, Meta, or Informational>
-category (*only required for Standards Track): <Core, Networking, Interface, or ERC>
+category: <Core, Networking, Interface, or ERC> # Only required for Standards Track. Otherwise, remove this field.
 created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
-requires (*optional): <EIP number(s)>
+requires: <EIP number(s)> # Only required when you reference an EIP in the `Specification` section. Otherwise, remove this field.
 ---
 
-This is the suggested template for new EIPs.
+<!--
+  READ EIP-1 (https://eips.ethereum.org/EIPS/eip-1) BEFORE USING THIS TEMPLATE!
 
-Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
+  This is the suggested template for new EIPs. After you have filled in the requisite fields, please delete these comments.
 
-The title should be 44 characters or less. It should not repeat the EIP number in title, irrespective of the category. 
+  Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
+
+  The title should be 44 characters or less. It should not repeat the EIP number in title, irrespective of the category.
+  
+  TODO: Remove this comment before submitting
+-->
 
 ## Abstract
-Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
 
-## Motivation
```

Comment:
Why not include the Motivation section? Most EIPs should have one, right?

Reply:
Most people give a long history of why an EIP is needed. Yes, most EIPs should have a Motivation section, but I think that most of them should be written after being merged as a Draft and before being moved to Review, after they get the hang of the process a bit.